source/testcommon/README.md

```diff
@@ -0,0 +1,9 @@
+# Test Common Py
+
+This is common testing utilities for Python projects.
```

Comment:
We need much more documentation on this project.
But I'm fine with adding it in another PR.
@ShaBloze do you have some ideas or experience on how and where we would do this?